node/src/components/rest_server.rs

```diff
@@ -78,6 +78,8 @@ impl<REv> ReactorEventT for REv where
 
 #[derive(DataSize, Debug)]
 pub(crate) struct RestServer {
+    /// Enable.
```

Comment:
Maybe: `Indicates whether the server is enabled or not`.
It may seem superfluous, but I like doc comments to be descriptive.

Up to you, though.

Reply:
Done.